rest_api/v2/components/index.yml

```
@@ -1613,6 +1664,28 @@ parameters:
       type: string
       enum: ['fr', 'es', 'it', 'en', 'de']
     description: Language in which the categories are going to be displayed
+  query_category_country:
+    in: query
+    name: country
+    required: false
+    description: Filter by country where categories are availables
+    schema:
+      type: string
+      enum: ['EN','FR', 'ES', 'IT', 'BR']
+  query_category_state:
+    in: query
+    name: state
+    required: false
+    schema:
+      type: string
+      enum: ['OK', 'DEPRECATED']
+  query_category_block_push_long:
+    in: query
+    name: block_push_long_description_on_gmb
+    description: Long description can't be set on GMB if a business own a category with this flag set to True
```

I am not sure we should describe what does `block_push_long_description_on_gmb` means here :)

```suggestion
    description: Filter on `block_push_long_description_on_gmb` param 
```

---

I was thinking it was not so bad to explain what this attribute means in a way this information can be useful for clients

---

I was thinking it was not so bad to explain what this attribute means in a way this information can be useful for clients

---

```suggestion
    description: Filter on `block_push_long_description_on_gmb` param (boolean indicating if the business long description can be pushed to GMB when the business belongs to this category)
```

---

Comme ça?

---

lets go 😄